tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
 
-    "target": "esnext",
+    "target": "es2019",
```

Comment:
Perhaps we should target a less recent is standard, @vito80ba what's your opinion?